<launch>
    <include file="$(find avoidance)/launch/global_planner_sitl_mavros.launch" >
        <arg name="use_three_point_msg" value="true" />
    </include>
    
    <node name="bezier_mavros_wrapper" pkg="offboard" type="bezier_mapping.py" output="screen">
        <param name="bezier_duration" value="1.5" />
        <param name="cross_factor" value="2.0" />
        <remap from="/path/bezier_pt" to="/three_point_path" />
    </node>
</launch>
